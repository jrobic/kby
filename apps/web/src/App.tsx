import { GraphQLClient } from "graphql-request";

import { Button } from "@kby/ui";

import { useHelloQuery } from "./generated/types-and-hooks";
import "./App.css";

const client = new GraphQLClient(process.env.KBY_API_GRAPHQL_URL as string, {});

export function App() {
  const { data } = useHelloQuery(client);

  return (
    <div className="App">
      <header className="App-header">
        <p>{data?.hello}</p>
        <Button label="Boop" primary size="small" />
      </header>
    </div>
  );
}
