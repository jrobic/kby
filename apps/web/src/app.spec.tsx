import { render } from "@testing-library/react";
import { App } from "./App";

test.skip("should render", () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
