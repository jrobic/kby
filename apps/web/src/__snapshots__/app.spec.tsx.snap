// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
        src="logo.svg"
      />
      <p>
        Hello Vite + React!
      </p>
      <p>
        <button
          type="button"
        >
          Boop
        </button>
        <button
          type="button"
        >
          count is: 0
        </button>
      </p>
      <p>
        Edit 
        <code>
          App.tsx
        </code>
         and save to test HMR updates.
      </p>
      <p>
        <a
          class="App-link"
          href="https://reactjs.org"
          rel="noopener noreferrer"
          target="_blank"
        >
          Learn React
        </a>
         | 
        <a
          class="App-link"
          href="https://vitejs.dev/guide/features.html"
          rel="noopener noreferrer"
          target="_blank"
        >
          Vite Docs
        </a>
      </p>
    </header>
  </div>
</DocumentFragment>
`;
